packages/material-ui/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -2,41 +2,97 @@ import * as React from 'react';
 import { isFragment } from 'react-is';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import withStyles from '../styles/withStyles';
+import { deepmerge } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import Typography from '../Typography';
 import BreadcrumbCollapsed from './BreadcrumbCollapsed';
```

Comment:
It's important that we convert the `BreadcrumbCollapsed` component with this PR, so that we don't miss it once this one is merged.